Source/WebCore/page/EventHandler.cpp

```diff
@@ -1978,6 +1978,43 @@ bool EventHandler::passMouseMovedEventToScrollbars(const PlatformMouseEvent& eve
     return handleMouseMoveEvent(event, &hitTestResult, true);
 }
 
+OptionSet<HitTestRequest::Type> EventHandler::getHitTypeForMouseMoveEvent(const PlatformMouseEvent& platformMouseEvent, bool onlyUpdateScrollbars)
+{
+    OptionSet<HitTestRequest::Type> hitType { HitTestRequest::Type::Move, HitTestRequest::Type::DisallowUserAgentShadowContent, HitTestRequest::Type::AllowFrameScrollbars };
```

Comment:
Nit - I think you can fall back on template type deduction here:

```suggestion
    OptionSet hitType { HitTestRequest::Type::Move, HitTestRequest::Type::DisallowUserAgentShadowContent, HitTestRequest::Type::AllowFrameScrollbars };
```

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/b7607875e50a6089c8262d0f679251a1a5216add